Casks/zeplin.rb

```diff
@@ -1,5 +1,5 @@
 cask "zeplin" do
-  version "4.1,1469"
+  version "7.4.0"
```

Comment:
```suggestion
  version "4.1.1,1472"
```

`Info.plist` in the latest version downloaded from the homepage agrees with the version as per `livecheck` as well as the `url` of the sparkle feed. `7.4.0` can be found in https://github.com/ScoopInstaller/Extras/blob/master/bucket/zeplin.json, though, so version numbers seem to differ for at least these two operating systems.

Reply:
This was `brew bump` having some fun with versions. It really should be using the livecheck version.